Sources/Caching/DeviceCache.swift

```diff
@@ -175,10 +175,6 @@ class DeviceCache {
         )
     }
 
-    func clearOfferingsCacheTimestamp() {
-        self.offeringsCachedObject.clearCacheTimestamp()
-    }
-
     func setOfferingsCacheTimestampToNow() {
```

Comment:
Hmm I was thinking, did we also change where we call this to only call it on the success case? Otherwise, if there is an error and we are calling this BEFORE the request happens, this change could cause some unintended behavior.

Reply:
Yeah this is only called by `preventPurchasePopupCallFromTriggeringCacheRefresh`.

Reply:
Which doesn't make a lot of sense btw, I filed SDK-3172 to address that.

Reply:
I removed that in #2623. Any other concerns here?